spec/controllers/v0/post911_gi_bill_statuses_controller_spec.rb

```diff
@@ -25,12 +25,6 @@
 
       gi_bill_200 = { cassette_name: 'evss/gi_bill_status/gi_bill_status' }
       context 'when EVSS response is 403', vcr: gi_bill_200 do
-        it 'has a response that matches the schema' do
-          get :show
-          expect(response).to have_http_status(:ok)
-          expect(response).to match_response_schema('post911_gi_bill_status', strict: false)
-        end
-
```

Comment:
Had to remove this test early, because it doesn't match the schema updates.  This whole spec file will be deleted in a future PR, so this is a temporary change.